style.css

```diff
@@ -125,3 +161,196 @@ button {
 .banner-scroll-btn button img {
   padding-top: 2px;
 }
+
+/* end hero section */
+
+/* work section */
+.works-section-title {
+  margin-top: 0;
+}
+
+.cards-container {
+  display: grid;
+  gap: 24px;
+  grid-template-columns: 1;
+}
+
+.work-card {
+  display: flex;
+  flex-direction: column;
+  background: var(--neutral-color);
+  border: 1px solid var(--secondary-color-50);
+  border-radius: 8px;
+  overflow: hidden;
+}
+
+.work-card__head {
+  height: 200px;
+}
+
+.work-card__body {
+  color: var(--secondary-color);
+  background: #fff;
+  border-radius: 8px 8px 0 0;
+}
+
+.card-title {
+  font-weight: 600;
+  font-size: 1.5rem;
+  line-height: 2rem;
+  text-align: center;
+  margin: 1rem;
+}
+
+.card-btns {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card-btn {
+  font-weight: 600;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  padding: 8px 12px;
+  background: var(--secondary-color-50);
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+}
+
+.cta-btn {
+  text-align: center;
+}
+
+.cta-btn a,
+.form-submit-btn {
+  display: inline-block;
+  color: #fff;
+  font-size: 1rem;
+  padding: 12px;
+  text-align: center;
+  margin: 24px auto;
+  border-radius: 4px;
+  background: var(--highlight-color);
+}
+
+/* end work section */
+
+/* about me section */
+.about-me-header {
+  position: relative;
+}
+
+.about-me-header-container {
+  display: flex;
+  flex-direction: column;
+  padding: 1.25rem;
+}
+
+.about-me-sub-title {
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  font-weight: 400;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.about-me-cards-container {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.about-me-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: var(--secondary-color-50);
+  border: 1px solid var(--neutral-color-50);
+  border-radius: 8px;
+  overflow: hidden;
+  padding: 24px;
+}
+
+.about-me-card__icon {
+  background: #fff;
+  margin: 24px 0;
+  padding: 18px;
+  width: 80px;
+  height: 80px;
+  border-radius: 200px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-shadow: 0 16px 24px var(--neutral-color);
+}
+
+.about-me-card__tech-stack {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 8px;
+}
+
+.about-me-card__tech-stack li {
+  color: var(--highlight-color);
+  font-weight: 600;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  padding: 12px;
+  background: #fff;
+  border-radius: 8px;
+}
+
+.about-me-header .illus-1 {
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.about-me-header .illus-2 {
+  position: absolute;
+  bottom: 48px;
+  left: 0;
+}
```

Comment:
The button in the `about me ` section should have the text `get my resume` as seen in the design given and not `see project` as seen in your design 

figma design :

![image](https://github.com/thasipablo/Personal-Portfolio/assets/20537712/a609ea0e-80bf-4caa-af54-0f0b2c12891e)

Your design :
![image](https://github.com/thasipablo/Personal-Portfolio/assets/20537712/00198e86-a946-42f3-99c6-e9ea04a1144d)